applications/FluidDynamicsApplication/fluid_dynamics_application_variables.cpp

```diff
@@ -39,10 +39,10 @@ KRATOS_CREATE_3D_VARIABLE_WITH_COMPONENTS(COARSE_VELOCITY)
 KRATOS_CREATE_VARIABLE(double,FIC_BETA)
 
 // Adjoint variables
-KRATOS_CREATE_3D_VARIABLE_WITH_COMPONENTS(ADJOINT_FLUID_VECTOR_1)
-KRATOS_CREATE_3D_VARIABLE_WITH_COMPONENTS(ADJOINT_FLUID_VECTOR_2)
-KRATOS_CREATE_3D_VARIABLE_WITH_COMPONENTS(ADJOINT_FLUID_VECTOR_3)
 KRATOS_CREATE_3D_VARIABLE_WITH_COMPONENTS(AUX_ADJOINT_FLUID_VECTOR_1)
+KRATOS_CREATE_3D_VARIABLE_WITH_COMPONENTS_WITH_TIME_DERIVATIVE(ADJOINT_FLUID_VECTOR_3, AUX_ADJOINT_FLUID_VECTOR_1);
+KRATOS_CREATE_3D_VARIABLE_WITH_COMPONENTS_WITH_TIME_DERIVATIVE(ADJOINT_FLUID_VECTOR_2, ADJOINT_FLUID_VECTOR_3);
+KRATOS_CREATE_3D_VARIABLE_WITH_COMPONENTS_WITH_TIME_DERIVATIVE(ADJOINT_FLUID_VECTOR_1, ADJOINT_FLUID_VECTOR_2);
```

Comment:
mmm... are these time derivatives?

Reply:
They are the adjoints of the time derivative equations (one after the other). So I would say they have a close relatinship with the time derivative.

Reply:
OK. Then fine to me (but providing a more descriptive name would help in this regard, not sure if it is possible...).

Reply:
Even I am also not sure because these are used in structural adjoint formulations as well. :/

Reply:
Removed.